fastapi[all]
mysql-connector-python==8.0.29 # ^ utf-8 error